CHANGELOG.md

```diff
@@ -1,26 +1,75 @@
 # Changelog
 
-## v0.8.0 (to be released)
+## [v0.8.0](https://github.com/FerretDB/FerretDB/releases/tag/v0.8.0) (2023-01-02)
 
 ### What's Changed
 
-In this release, we made two big changes in the way FerretDB stores data in PostgreSQL.
+We are pleased to announce our first Beta release!
 
-The first change is about the format we use to store documents.
+#### Storage changes for PostgreSQL
 
-Previously, we were storing information about data types in the fields themselves.
-Starting from this release, we store information about data types (document's schema) in a special field.
+We made a few _backward-incompatible_ changes in the way we store data in PostgreSQL to improve FerretDB performance.
+In the future, those changes will allow us to better use indexes and query collections faster.
 
-This will allow us to implement more query push downs in the future.
+To keep your data:
 
-The second change is about the way FerretDB stores metadata about the collections.
+* backup databases using `mongodump` or `mongoexport`;
+* stop FerretDB;
+* drop PostgreSQL views for FerretDB databases;
```

Comment:
Just suggestion but imo we could _recommend_ to dump the postgresql database either, just in case if there would be any issues that we don't know about (highly impossible)